lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -67,21 +67,25 @@ def sort_predicate
           @sort_predicate ||= search_config.results_sort_predicate
         end
 
-        def convert_search_to_json(results)
+        def convert_results_to_json(results)
           json_results = []
-          results.each do |result|
-            uri = result[:uri].first.to_s
-            id = result[:id].first.to_s
-            label = language_sort_service.new(result[:label], language).sort
-            altlabel = language_sort_service.new(result[:altlabel], language).sort
-            json_results << { uri: uri, id: id, label: full_label(label, altlabel) }
-          end
+          results.each { |result| json_results << convert_result_to_json(result) }
```

Comment:
Split out construction of the json hash to make it easier in the next PR to add context to the result.  There is no change in behavior.